compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -569,6 +570,139 @@ OMR::CodeGenerator::setUpForInstructionSelection()
 
   }
 
+/**
+ * TODO: Remove this function once OMR provides a proper API query function.
+ * OMR issue https://github.com/eclipse/omr/issues/842
```

Comment:
This query is no longer used. After reviewing all ILOpCodes that has the `Call` property, the existing `isFunctionCall` query is the proper query to use.